Kudu.Services.Web/App_Start/NinjectServices.cs

```diff
@@ -720,11 +724,24 @@ private static IEnvironment GetEnvironment(IDeploymentSettingsManager settings =
             string root = PathResolver.ResolveRootPath();
             string siteRoot = Path.Combine(root, Constants.SiteFolder);
             string repositoryPath = Path.Combine(siteRoot, settings == null ? Constants.RepositoryPath : settings.GetRepositoryPath());
+            string binPath = HttpRuntime.BinDirectory;
+
+            if (!string.IsNullOrWhiteSpace(binPath) && !OSDetecter.IsOnWindows())
+            {
+                int binIdx = binPath.LastIndexOf("Bin", StringComparison.Ordinal);
```

Comment:
yes, i do EndWIth at line 736. use LastIndexOf is to make sure i cover both "bin" and "bin/" cases